README.md

```diff
@@ -647,6 +649,14 @@ podman restart konflux-control-plane
 **Note:** It might take a few minutes for the UI to become available once the container
 is restarted.
 
+#### Cluster lacks deault storage
+The `deploy-deps` script includes a check to verify whether the cluster has default storage configured.
+If default storage is absent, the script will fail, displaying: 
+`error: timed out waiting for the condition on persistentvolumeclaims/test-pvc`.
+
+If you are using Kind, try to re-create the cluster. 
+Otherwise, ensure that PVCs (Persistent Volume Claims) can be allocated for the cluster's default storage class.
```

Comment:
Let's introduce line breaks at 89 characters (same for other lines in this PR as well).

Reply:
fixed